paasta_tools/cli/cmds/push_to_registry.py

```diff
@@ -79,72 +78,105 @@ def add_subparser(subparsers):
         "-f",
         "--force",
         help=(
-            "Do not check if the image is already in the PaaSTA docker registry. "
+            "Do not check if the image is already in the PaaSTA docker registries. "
             "Push it anyway."
         ),
         action="store_true",
     )
+    list_parser.add_argument(
+        "-r",
+        "--registries",
+        help=(
+            "Docker Registry URLs, such as docker-paasta-norcal.yelpcorp.com:443, separated by semicolons."
+            "Restricts the Docker Registry URLs to only a specified list."
+        ),
+    )
     list_parser.set_defaults(command=paasta_push_to_registry)
 
 
-def build_command(upstream_job_name, upstream_git_commit):
+def build_commands(upstream_job_name, upstream_git_commit):
     # This is kinda dumb since we just cleaned the 'services-' off of the
     # service so we could validate it, but the Docker image will have the full
     # name with 'services-' so add it back.
-    tag = build_docker_tag(upstream_job_name, upstream_git_commit)
-    cmd = f"docker push {tag}"
-    return cmd
+    tags = build_docker_tags(upstream_job_name, upstream_git_commit)
+    cmds = [f"docker push {tag}" for tag in tags]
+    return cmds
 
 
 def paasta_push_to_registry(args):
-    """Upload a docker image to a registry"""
+    """Upload a docker image to registries"""
     service = args.service
     if service and service.startswith("services-"):
         service = service.split("services-", 1)[1]
     validate_service_name(service, args.soa_dir)
 
-    if not args.force:
-        try:
-            if is_docker_image_already_in_registry(service, args.soa_dir, args.commit):
-                paasta_print(
-                    "The docker image is already in the PaaSTA docker registry. "
-                    "I'm NOT overriding the existing image. "
-                    "Add --force to override the image in the registry if you are sure what you are doing."
-                )
-                return 0
-        except RequestException as e:
-            registry_uri = get_service_docker_registry(service, args.soa_dir)
+    registries = None
+
+    if args.registries:
+        registries = args.registries.split(";")
+
+    try:
+        image_in_registry_or_not = where_does_docker_image_exist_and_does_not(
```

Comment:
don't you need to pass the argument down here? otherwise it'll look up in a different place to the argument?

Reply:
Can you explain this better? I didn't quite understand.

Reply:
Addressed.